easyaudit/migrations/0019_alter_crudevent_changed_fields_and_more.py

```diff
@@ -0,0 +1,53 @@
+# Generated by Django 5.0.1 on 2024-03-18 19:53
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('easyaudit', '0018_rename_crudevent_object_id_content_type_index'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='crudevent',
+            name='changed_fields',
+            field=models.TextField(blank=True, default='', verbose_name='Changed fields'),
+        ),
+        migrations.AlterField(
+            model_name='crudevent',
+            name='object_json_repr',
+            field=models.TextField(blank=True, default='', verbose_name='Object JSON representation'),
+        ),
+        migrations.AlterField(
+            model_name='crudevent',
+            name='object_repr',
+            field=models.TextField(blank=True, default='', verbose_name='Object representation'),
+        ),
+        migrations.AlterField(
+            model_name='crudevent',
+            name='user_pk_as_string',
```

Comment:
22679173 items with null value on prod

Reply:
@klplentific that's the biggest, 22M + 5M.
I think it make sense to prepare management command (or run by shell) to migrate `null` -> `""` by 100K - 0.5M per iteration with small delays..

And after that deploy code to make the migration.